src/test/smart-components/order/orders.test.js

```diff
@@ -85,11 +98,39 @@ describe('<Orders />', () => {
     const wrapper = mount(<ComponentWrapper store={ store }><Orders { ...initialProps } /></ComponentWrapper>);
     setImmediate(() => {
       wrapper.update();
-      expect(wrapper.find(DataListContent).props().isHidden).toEqual(true);
+      expect(wrapper.find(DataListContent).first().props().isHidden).toEqual(true);
       wrapper.find('.pf-c-data-list__toggle').first().simulate('click');
-      expect(wrapper.find(DataListContent).props().isHidden).toEqual(false);
+      expect(wrapper.find(DataListContent).first().props().isHidden).toEqual(false);
       expect(wrapper.find(OrderDetailTable)).toHaveLength(1);
       done();
     });
   });
+
+  it('should render past orders correctly', () => {
+    initialState = { orderReducer: { ...orderInitialState, linkedOrders, isLoading: false },
+      portfolioReducer: { ...portfoliosInitialState, isLoading: false }};
+    const store = mockStore(initialState);
+    const wrapper = shallow(<Orders store={ store } { ...initialProps } />);
+    expect(shallowToJson(wrapper)).toMatchSnapshot();
+  });
+
+  it('should set the Manage Product link to open in a new tab', (done) => {
+    const store = mockStore({ ...initialState, orderReducer: { ...initialState.orderReducer, linkedOrders }});
+    apiClientMock.get(`${CATALOG_API_BASE}/orders`, mockOnce({ body: { data: []}}));
+    apiClientMock.get(`${APPROVAL_API_BASE}/requests`, mockOnce({ body: { data: []}}));
+    apiClientMock.get(`${CATALOG_API_BASE}/portfolio_items`, mockOnce({ body: { data: []}}));
+    apiClientMock.get(`${CATALOG_API_BASE}/order_items`, mockOnce({ body: { data: []}}));
+
+    const wrapper = mount(<ComponentWrapper store={ store }><Orders { ...initialProps } linkedOrders /></ComponentWrapper>);
+    setImmediate(() => {
+      wrapper.update();
+      expect(wrapper.find(DataListContent).first().props().isHidden).toEqual(true);
+      expect(wrapper.find(DataListCell).last().props().children.props.children.props).toMatchObject(
```

Comment:
You can also call `find` on enzyme nodes so you dont have to chain the children calls. Something like
```javascript
expect(wrapper.find(DataListCell).last().find('a').props()).to....
```
It should return the same result.

Reply:
done